src/dapp/DSRuneAbstract.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.12;
+
+// https://github.com/makerdao/dss-spellbook
+contract DSRuneAbstract {
+    function pause()    public view returns (address);
+    function usr()      public view returns (address);
```

Comment:
Just confirming: The `usr` is the address that was formerly called `action` (the contract that contains the action logic), right?

Also, if these are getter functions to public variables, I believe the compiler (v0.5.3+) automatically creates getter functions for all public state variables.

Reply:
@brianmcmichael Ah, just read your above comment on `action`  👍

Reply:
Hi Kenton,

We will need to define them in the spec so that implementers make sure to include them in their contracts. Ideally these would go in to the contract as public variables, we just want to ensure that they will have the same names and are present.

Still up in the air on whether or which of `usr` or `action` to include here.